app/Http/Controllers/Api/V1/ProjectController.php

```diff
@@ -724,8 +749,10 @@ public function favorite(Request $request, Organization $suborganization, Projec
      * Get All Favorite Projects of login user
      *
      * Get a list of the favorite projects of a user.
-     * @param Request $request
+     * 
      * @urlParam Organization $suborganization
+     * 
+     * @param Request $request
```

Comment:
Need to move "@responseFile" before "@param".

Reply:
Done!